src/community/index.md

```diff
@@ -5,67 +5,64 @@ title: "Community and Support"
 description: "Communities, mailing lists, and bug databases for the Dart project."
 ---
 
-Track the Dart project and join the conversation in a variety of ways.
+Track the Dart project, get help, and talk with other Dart developers.
 
-Also see:
+## Stay informed
 
-* [Community and Support]({{site.webdev}}/community) on Dart webdev
-* [Dart Academy](https://dart.academy/), which contains a collection of tutorials
-  and articles written by the Dart community
+* [news.dartlang.org](http://news.dartlang.org) -
+  Official news feed for the Dart project.
+* [announce@dartlang.org](https://groups.google.com/a/dartlang.org/d/forum/announce) -
+  Low traffic announcements of new releases, breaking changes,
+  and other important news. Recommended!
+* [@dart_lang](https://twitter.com/dart_lang) -
+  The official Twitter account.
+* [+dartlang](https://plus.google.com/+dartlang) -
+  The official Google+ account.
 
 ## Join the conversation
 
-Dart is an open-source project, and we look forward to hearing from you.
-
-#### General
-
-* [Discussion](https://groups.google.com/a/dartlang.org/d/forum/misc)
-* [Google+ page](https://plus.google.com/+dartlang)
-* [Twitter](https://twitter.com/dart_lang)
-* [Bugs and feature requests](https://github.com/dart-lang/sdk/issues/new)
+Get answers and connect with Dart developers.
 
 #### Communities
 
-* [StackOverflow](http://stackoverflow.com/tags/dart)
-* [Reddit](https://www.reddit.com/r/dartlang)
+* [StackOverflow](http://stackoverflow.com/tags/dart) -
+  The best place for how-to questions.
+* [Gitter](https://gitter.im/dart-lang) -
```

Comment:
Yikes, I think this should be https://gitter.im/dart-lang/home.

@kevmoo, could you change it?